src/common/utils.ts

```diff
@@ -9,6 +9,7 @@ import dayjs from 'dayjs';
 import * as relativeTime from 'dayjs/plugin/relativeTime';
 import * as updateLocale from 'dayjs/plugin/updateLocale';
 import { Disposable, Event, Uri } from 'vscode';
+// TODO: localization for webview needed
```

Comment:
Does this file get used by webviews? If so, how does that `vscode` import work?

Reply:
It does, but we're only using the types from `vscode`, so it's fine today (added the explicit `type` to the import).